code/game/objects/items/weapons/tools/misc.dm

```diff
@@ -179,3 +179,46 @@
 	desc = "A small metal two pronged fork used to gather balls of silk."
 	icon_state = "silk_wand"
 	tool_qualities = list(QUALITY_WEAVING = 30)
+
+
+/obj/item/tool/multitool_improvised
+	name= "improvised multitool implant"
+	desc = "A jury-rigged implant, holding cobbled-together tools. For those who are more interested in tool carrying than scared of tetanus."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "multitool_improvised"
+	force = WEAPON_FORCE_PAINFUL
+	switched_on_force = WEAPON_FORCE_PAINFUL * 0.8
+	worksound = WORKSOUND_DRIVER_TOOL
+	flags = CONDUCT
+	switched_on_qualities = list(
+		QUALITY_CUTTING = 15,
+		QUALITY_DRILLING = 5,
+		QUALITY_SCREW_DRIVING = 15,
+		QUALITY_WIRE_CUTTING = 20,
+		QUALITY_RETRACTING = 10,
+		QUALITY_BONE_SETTING = 10,
+		QUALITY_PRYING = 10,
+		QUALITY_HAMMERING = 10,
```

Comment:
```suggestion
		QUALITY_HAMMERING = 20,
```
Baseline standardization for crafting recipes. Considering makeshift has half the values than normal tools, this actually works for certain crafting recipes, making it not a literal downgraded homewrecker.